functions-and-scope.js

```diff
@@ -50,15 +93,36 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomsten:
 // averageGrade(grades) geeft 6.642857142857143
-// averageGrade([6, 4, 5]) geeft xxxx
-// averageGrade([8, 9, 4, 6, 10]) geeft xxxx
+// averageGrade([6, 4, 5]) geeft 5
+// averageGrade([8, 9, 4, 6, 10]) geeft 7.4
 
+function averageGrade (gradeList) {
+    let totalGrade = null;
+    let averageGradeList = null;
+    for (let i = 0; i < gradeList.length; i++) {
+        totalGrade += gradeList[i];
+        averageGradeList = totalGrade / gradeList.length;
+    }
+    return averageGradeList;
+}
+
+console.log(averageGrade(grades));
 
 /* 2c: Afronden op twee decimalen */
 // Zorg ervoor dat het gemiddelde cijfer dat wordt teruggegeven uit de functie netjes wordt afgerond op twee decimalen.
 // Tip: Google is your best friend!
 
+function averageGradeRound (gradeList) {
+    let totalGrade = null;
+    let averageGradeList = null;
+    for (let i = 0; i < gradeList.length; i++) {
+        totalGrade += gradeList[i];
+        averageGradeList = totalGrade / gradeList.length;
+    }
+    return Math.round(averageGradeList * 100) / 100;
```

Comment:
Nice! 

( kuch er is trouwens ook een methode `.toFixed` waar je fracties mee kan geven kuch 😶 )